CMS-Enterprise/mint-app
# [MINT-3247] Move subcategory action

# MINT-3247

## Description


- Added modal to table dropdown for moving a subcategory
- Migrated custom category form to use `useFormatMTOCategories`
- Added disabled states to dropdown form uncategorized categories
- Added `MTOModalType` for growing number of types
- Unit test


## How to test this change

1.  Verify subcategories accurately reflect the current category and switch correctly
2. Figma matches

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
